index.html

```diff
@@ -0,0 +1,257 @@
+                <a href="#"><img class="icon-image" src="./Assets/logos/Vector.png" alt="" /></a>
+            </li>
+            <li class="icon_link">
+                <a href="#"><img class="icon-image" src="./Assets/logos/linkin.png" alt="" /></a>
+            </li>
+            <li class="icon_link">
+                <a href="#"><img class="icon-image" src="./Assets/logos/fingers.png" alt="" /></a>
+            </li>
+            <li class="icon_link">
+                <a href="#"><img class="icon-image" src="./Assets/logos/twitter.png" alt="" /></a>
+            </li>
+            <li class="icon_link">
+                <a href="#"><img class="icon-image" src="./Assets/logos/letterM.png" alt="" /></a>
+            </li>
+        </ul>
+    </section>
+
+    <!-- MIDDLE SECTION -->
+    <setion class="middlesection">
+        <div class="middlesection-container">
+            <h2 class="middlesection_header_title">My Recent Works</h2>
+
+            <hr />
+            <img class="middlesection-image" src="./Assets/images/middlesectionimage.png" alt="" />
+        </div>
+    </setion>
+    <!-- MULTIPLES STORIES -->
+
+    <section class="multiples-stories">
+        <div class="multiples-stories_container">
+            <h2 class="multiples-stories_header">Multi-Post Stories</h2>
+            <p class="multiples-stories_paragraph">
+                A daily selection of privately personalized reads; no accounts or
+                sign-ups required. has been the industry's standard dummy text ever
+                since the 1500s, when an unknown printer took a standard dummy text.
+            </p>
+            <ul class="button-flex">
+                <li><button class="default-btn">CSS</button></li>
+                <li><button class="default-btn">HTML</button></li>
+                <li><button class="default-btn">Bootstrap</button></li>
+                <li><button class="default-btn">Ruby</button></li>
+            </ul>
+            <div class="multiple-btn-container">
+                <button class="primarybtn">See Project</button>
+            </div>
+        </div>
+    </section>
+
+    <!-- PROFESSION ART PRINTING -->
+    <section class="professional_art_printing-section">
+        <div class="professional_art_printing-section-items">
+            <div class="art_print_container">
+                <h2 class="art_print_header">Profesional Art Printing Data</h2>
+                <p class="art_print_paragraph">
+                    A daily selection of privately personalized reads; no accounts or
+                    sign-ups required. Has been the industry's standard.
+                </p>
+                <ul class="art_print_btn_container">
+                    <li><button class="transparent-btn">HTML</button></li>
+                    <li><button class="transparent-btn">Bootstrap</button></li>
+                    <li><button class="transparent-btn">Ruby</button></li>
+                </ul>
+            </div>
```

Comment:
- [ ] Please consider using a flex-box to position elements inside the work section elements. That mean `art_print_container` should use flex-box to position elements inside it. This will make your project way more maintainable as fex-box is an incredible 💪 😮 tool for responsiveness 😄